src/Cookie/FileCookieJar.php

```diff
@@ -88,7 +89,7 @@ public function load(string $filename): void
             foreach ($data as $cookie) {
                 $this->setCookie(new SetCookie($cookie));
             }
-        } elseif (\strlen($data)) {
+        } elseif (is_scalar($data) && !empty($data)) {
```

Comment:
Missing leading slash.

Reply:
Wouldn't this be a matter of configuring CS-Fixer to prevent future review efforts? Or what about importing these methods?

Fixed in 3f37ada3b1af0d7d027b165cc494c5269c3da9e3